### DISCUSSION TITLE
"Dependency Lookup Warnings" in PR

### DISCUSSION THREAD

**Discussion by muffl0n** [renovatebot/renovate#23236] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.1.16

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Details for "Dependency Lookup Warnings" in PRs or at least message on level "ERROR" in log

### What you tried so far.

Currently, all my PRs in Bitbucket Server show this warning
<img width="790" alt="screenshot 2023-07-07 um 11 45 42" src="https://github.com/renovatebot/renovate/assets/927904/fcc988f8-b2e7-490b-8ff6-2fb6de5bca50">

Unfortunately, the error is only logged on level "DEBUG" which makes it hard to find cause we have to change the log level each time this occurs:
```
DEBUG: Found package lookup warnings (repository=project/webapp, branch=renovate/major-maven-surefire-plugin.version)
         "warnings": [
           "Failed to look up maven package mypackage:webapp-core",
           "Failed to look up maven package mypackage:webapp-common"
         ],
         "warningFiles": ["webapps/common/pom.xml", "webapps/app/pom.xml", "webapps/default/pom.xml"]
```

It would be awesome, if the warning would be written to the PR itself. Is there an option for that?
Also, the output of this warning at the DEBUG level feels wrong somehow.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
We recommend that you run with DEBUG logging. For example this is what we do in the hosted app

**Reply by z0rc** [reply]
We run self-hosted renovate on self-hosted gitlab. We have tens of repos, renovate debug logs for single run is around 12 Mb. It's tedious process to dig through this file in order to get more details about these warnings. Also this requires renovate operational knowledge, which most consumers of renovate MRs do not have.

**Reply by rarkins** [reply]
We host over 600k repos and manage debug logging fine

**Reply by muffl0n** [reply]
I can understand, that it's easy to spot those errors if you have sophisticated log analyzing. We do not have that (yet), cause we just started.
Running an application with log level DEBUG does not feel right to me. As you can see, as a new user I'd expect errors to be logged on ERROR, not on DEBUG. I'm sure I'm not the only newbie confused by that.

If there's no intention to change that, I guess I have to setup log analyzing.

Thank you very much for your assistance and your awesome project!

**Reply by muffl0n** [reply]
Seems like I was too blind. There is indeed a log message on level `WARN`
```
   WARN: Package lookup failures (repository=foo/bar, branch=renovate/ms-teams-webhook-2.x)
         "warnings": [
           "Failed to look up maven package org.jetbrains.teamcity:configs-dsl-kotlin-parent",
           "Failed to look up maven package org.jetbrains.teamcity:configs-dsl-kotlin-plugins"
         ],
         "files": [".teamcity/pom.xml"]
```
